.buildkite/pipeline.yml

```
@@ -1,7 +1,7 @@
 # Nodes with values to reuse in the pipeline.
 common_params:
   plugins: &common_plugins
-  - &bash_cache automattic/bash-cache#2.10.0
+  - &bash_cache automattic/a8c-ci-toolkit#2.13.0
```

I don't see the `&bash_cache` anchor being referenced as an alias. So we might as well get rid of it.

```suggestion
    - automattic/a8c-ci-toolkit#2.13.0
```

---

Good point, thanks!